src/main/webapp/app/course/manage/overview/course-management-card.scss

```diff
@@ -6,6 +6,13 @@
     }
 
     .card-heading {
+        opacity: 1;
+        filter: alpha(opacity = 100);
+        &:hover {
+            transition: 0.5s;
```

Comment:
You can move `transition: 0.5s;` into `.card-heading { HERE }`instead of the `&:hover` pseudo selector.

Also there is a SCA issue as you need to add a linebreak on line 11

Otherwise great job! Looks good